meveo-model/src/main/java/org/meveo/model/security/Role.java

```diff
@@ -37,6 +39,7 @@
  */
 @Entity
 @Cacheable
+@ModuleItem(path = "Role", value = "Roles")
```

Comment:
Here, same logic apply. It should be path = "roles" & value = "Role

Reply:
rectified

Reply:
@Fhadk I don't see any changes - did you commit ?